src/views/microworldshomepage/microworldshomepage.jsx

```diff
@@ -0,0 +1,59 @@
+var React = require('react');
+var injectIntl = require('react-intl').injectIntl;
+var FormattedHTMLMessage = require('react-intl').FormattedHTMLMessage;
+var FormattedMessage = require('react-intl').FormattedMessage;
+var render = require('../../lib/render.jsx');
+
+var MasonryGrid = require('../../components/masonrygrid/masonrygrid.jsx');
+var Page = require('../../components/page/www/page.jsx');
+var TitleBanner = require('../../components/title-banner/title-banner.jsx');
+var TTTTile = require('../../components/ttt-tile/ttt-tile.jsx');
+var Tiles = require('./microworlds.json');
+
+require('./microworldshomepage.scss');
+
+var MicroworldsHomepage = injectIntl(React.createClass({
+    type: 'MicroworldsHomepage',
+    renderTTTTiles: function () {
+        var formatMessage = this.props.intl.formatMessage;
+        var translatedTiles = [];
+        var translatedTile = {};
+
+        Tiles.map(function (tile, key) {
+            translatedTile = {
+                title: formatMessage({id: tile.title}),
+                tutorialLoc: tile.tutorialLoc,
+                thumbUrl: tile.thumbUrl,
+                bannerUrl: tile.bannerUrl
+            };
+            translatedTiles.push(
+                <TTTTile
+                    key={key}
+                    {...translatedTile}
+                />
+            );
+        }, this); // don't forget to pass 'this' into map function
+        return translatedTiles;
```

Comment:
@rschamp this is based on what is in the ttt view. However, I realized that the render was originally created to do the reordering of tiles based on the number of columns, then that got moved to the masonry grid component. So this probably should be merged into `render` as should the corresponding code in ttt.